holoviews/tests/operation/test_datashader.py

```diff
@@ -1530,3 +1531,17 @@ def test_uint64_dtype():
     curve = Curve(df)
     with pytest.raises(TypeError, match="Dtype of uint64 for column A is not supported."):
         rasterize(curve, dynamic=False, height=10, width=10)
+
+
+def test_imagestack_datashader_color_key():
+    d = np.arange(23)
+    df = pd.DataFrame({"x": d, "y": d, "language": list(map(str, d))})
+    points = Points(df, ["x", "y"], ["language"])
+
+    # This will run rasterize which outputs an ImageStack
+    op = datashade(
```

Comment:
I don't get why `datashade` is the operation used here instead of `rasterize` directly - `shade` means server-side colormixing which returns an RGB. Why not test that `rasterize` accepts `color_key` more directly?